scipy/stats/_multivariate.py

```diff
@@ -4081,7 +4081,7 @@ def rvs(self, loc=None, shape=1, df=1, size=1, random_state=None):
             x = rng.chisquare(df, size=size) / df
 
         z = rng.multivariate_normal(np.zeros(dim), shape, size=size)
-        samples = loc + z / np.sqrt(x)[:, None]
+        samples = loc + z / np.sqrt(x)[..., None]
```

Comment:
Will it be a good idea to wrap `x` as `np.atleast_1d(x)` to make sure ellipisis work if size is None?

Reply:
It works for me without doing that:

```py
>>> from scipy.stats import multivariate_t
>>> multivariate_t.rvs(size=None)
-2.212999544334533
```

Reply:
Ah. I was of the view that `x` will be a float as the output of `np.sqrt(x)`, but it turns out it is of `numpy.float64` type and ellipsis worked. 

> So, do you think we would need to wrap x in np.atleast_1d? 

Since it is working with `size=None`, there is no need now. Thanks for clarifying.

Reply:
> Since it is working with `size=None`, there is no need now.

Makes sense. Thanks for the review though